pkg/epforwarder/epforwarder.go

```diff
@@ -42,7 +42,7 @@ var passthroughPipelineDescs = []passthroughPipelineDesc{
 	{
 		eventType:              eventTypeDBMSamples,
 		endpointsConfigPrefix:  "database_monitoring.samples.",
-		hostnameEndpointPrefix: "dbquery-intake.",
+		hostnameEndpointPrefix: "dbm-metrics-intake.",
```

Comment:
This feels like a more important change than changing log messages, maybe worth a releasenote, but should at least be called out in the PR title and description.

Reply:
For context, this is a very low risk change... they both resolve to the same IP address so just cleaning up tech debt (probably don't want to confused customers by making a release note for this). I agree it's worth adding to the PR description though.

Reply:
I ended up removing this - this change caused deploys to the dev environment to fail so I wasn't able to test it. I'll make a Jira ticket to track this as a change we should make in the future.